packages/core/core/src/Transformation.js

```diff
@@ -123,18 +127,17 @@ export default class Transformation {
       try {
         await asset.loadExistingSourcemap();
       } catch (err) {
-        logger.warn([
+        logger.verbose([
           {
             origin: '@parcel/core',
-            message: `Could not load existing source map for ${relativePath(
-              this.options.projectRoot,
-              asset.value.filePath,
+            message: `Could not load existing source map for ${escapeMarkdown(
+              relativePath(this.options.projectRoot, asset.value.filePath),
```

Comment:
So `relativePath` always uses the unix-style, so no backslashes on Windows. Do we want that? or rather `path.relative`?

Reply:
Currently `relativePath` defaults to prepending `./` to the beginning of the path. I'm not sure of the value here for that either.